src/Workspaces/Core/Portable/Shared/Extensions/ProjectExtensions.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.CodeAnalysis.Shared.Extensions
     internal static partial class ProjectExtensions
     {
         public static bool IsFromPrimaryBranch(this Project project)
```

Comment:
>IsFromPrimaryBranch [](http://example.com/codeflow?start=27&length=19)

The extension method doesn't seem that useful now.

Reply:
trying to not break any semantics here :)

Reply:
Inlining this wouldn't break any.

Reply:
oh.  i see what you mean.  that's just not really something i'm concerned about right now :)  i'm just trying first and foremost to decouple code from jumping from solution-to-workspace, particularly in the remote case.